dbms/src/TiDB/Schema/TiDB.cpp

```diff
@@ -763,16 +765,10 @@ catch (const Poco::Exception & e)
 ///////////////////////
 
 IndexInfo::IndexInfo(Poco::JSON::Object::Ptr json)
-    : id(0)
-    , state(TiDB::SchemaState::StateNone)
-    , index_type(0)
-    , is_unique(true)
-    , is_primary(true)
-    , is_invisible(true)
-    , is_global(true)
```

Comment:
Remove because it is conflict with those defined in TiDB.h
https://github.com/pingcap/tiflash/blob/20e5abfa71e813b5e368fe03de4d9326c9fffcb5/dbms/src/TiDB/Schema/TiDB.h#L241-L260